services/raycast/installs.service.js

```diff
@@ -0,0 +1,68 @@
+import Joi from 'joi'
+import { metric } from '../text-formatters.js'
+import { anyInteger } from '../validators.js'
+import { BaseJsonService, NotFound, pathParams } from '../index.js'
+
+const schema = Joi.object({
+  download_count: anyInteger,
+}).required()
+
+export default class RaycastInstalls extends BaseJsonService {
+  static category = 'platform-support'
+
+  static route = {
+    base: 'raycast/installs',
```

Comment:
We seem to be mixing and matching a bit on "downloads" and "installs". The number we're getting from the API is `download_count`. Given that, I think we should frame this as a "downloads" badge (rather than "installs") and make the route badge `raycast/dt` in line with the conventions: https://github.com/badges/shields/blob/master/doc/badge-urls.md